src/team.hpp

```diff
@@ -180,7 +179,8 @@ class team
 	int side() const { return info_.side; }
 	int gold() const { return info_.gold; }
 	int start_gold() const { return info_.start_gold; }
-	int base_income() const { return info_.income + game_config::base_income; }
+	int raw_income() const { return info_.income; }
```

Comment:
Could you add a doc comment here regarding the difference between this and `base_income`.